subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/InProcessGradleExecuter.java

```diff
@@ -255,12 +250,7 @@ public JavaExecHandleBuilder create() {
     }
 
     private File getClasspathManifestJarFor(Collection<File> classpath) {
-        String cpString = CollectionUtils.join(" ", CollectionUtils.collect(classpath, new Transformer<String, File>() {
-            @Override
-            public String transform(File file) {
-                return file.toURI().toString();
-            }
-        }));
+        String cpString = CollectionUtils.join(" ", CollectionUtils.collect(classpath, file -> file.toURI().toString()));
```

Comment:
```suggestion
        String cpString = classpath.stream().map(File::toURI).map(URI::toString).collect(Collectors.joining(" "));
```

I would prefer Java 8 Stream. Your call.

Reply:
Using stream now.